refinery/analysis_manager/utils.py

```diff
@@ -20,176 +20,45 @@
 
 logger = logging.getLogger(__name__)
 
-# Allow JSON Schema to find the JSON pointers we define in our schemas
-JSON_SCHEMA_FILE_RESOLVER = RefResolver(
```

Comment:
If you kept this, could pieces like the UUID RE be defined separately? Would that be useful?